android/test_app/app/build.gradle

```diff
@@ -149,8 +149,8 @@ dependencies {
     //nativeBuildImplementation(name: 'pytorch_android_torchvision-release', ext: 'aar')
     //extractForNativeBuild(name: 'pytorch_android-release', ext: 'aar')
 
-    nightlyImplementation 'org.pytorch:pytorch_android:1.7.0-SNAPSHOT'
```

Comment:
do we need to bump version in `android/gradle.properties` ?

Reply:
Good catch on that one as well, updating